src/blueprints/timeline-state-resolver-types/atem-types.ts

```diff
@@ -0,0 +1,26 @@
+import { DeviceType } from '../../model/enums/device-type'
+
+// These values are taken from TSR TODO: Find a better way for types.
+export interface AtemTimelineObjectContent {
+  deviceType: DeviceType.ATEM,
+  type: AtemType,
+  me?: {
+    input?: number,
+    previewInput?: number
+  },
+  aux?: {
+    input: number
+  }
+}
+
+// Taken from TSR, so we must have the same values. // TODO: Find a better way to get the types from TSR
```

Comment:
Perhabs TSR and the Blueprints Repository needs a shared submodule, wherein the types are defined, and thus shared?